db/migrate/20200914005000_add_version_and_cloud_provider_to_configuration_profiles.rb

```diff
@@ -0,0 +1,6 @@
+class AddVersionAndCloudProviderToConfigurationProfiles < ActiveRecord::Migration[5.2]
+  def change
+    add_column :configuration_profiles, :default_version, :string
+    add_column :configuration_profiles, :cloud_provider, :string
```

Comment:
I think `cloud_provider` is going to be confusing because we have Cloud Providers in MIQ along with Infra Providers and others

I think something like "target_platform" would be better

Reply:
OK, it makes sense.